fvm/evm/handler/handler.go

```diff
@@ -224,6 +228,8 @@ func (h *ContractHandler) batchRun(rlpEncodedTxs [][]byte, coinbase types.Addres
 		if err != nil {
 			return nil, err
 		}
+
+		h.tracer.Collect(r.TxHash)
```

Comment:
Since this is not producing any error but the emit event might still fail, we might do it at last thing before return of function

Reply:
good call